lib/pyszn/parser.py

```diff
@@ -314,6 +317,13 @@ def find_topology_in_python(filename):
                 continue
             if node.targets[0].id == 'TOPOLOGY':
                 return node.value.s
+            elif node.targets[0].id == 'TOPOLOGY_ID':
+                topology_id = node.value.s
+                for search_path in base_szn_dir:
+                    for filename in glob.glob('{}/{}.szn'.format(search_path, topology_id)):
```

Comment:
@carlos-jenkins I wonder if this idea of mine is overkill:
```suggestion
                    for filename in glob.glob(str(Path(search_path) / '{}.szn'.format(topology_id))):
```